pkg/mock/client.go

```diff
@@ -30,6 +30,16 @@ func (hms *Client) AddExpectation(exp *Expectation) {
 	hms.callMock("expectation", string(msg))
 }
 
+func (hms *Client) Verify(v *Verification) {
```

Comment:
It would be nice to have same approach with functional options for verifications as for expectations to make it uniform and also provide safe defaults (i.e. atMost: -1)